accesshandler/pkg/server/routes.go

```diff
@@ -21,7 +20,6 @@ func (s *Server) Routes() http.Handler {
 	r.Use(chiMiddleware.RequestID)
 	r.Use(chiMiddleware.RealIP)
 	r.Use(chiMiddleware.Recoverer)
-	r.Use(chiMiddleware.Timeout(30 * time.Second))
```

Comment:
Is this not needed because the lambda timeout supersedes this one?

Reply:
yes